sessionctx/stmtctx/stmtctx.go

```diff
@@ -132,6 +132,22 @@ type StatementContext struct {
 	Tables []TableEntry
 }
 
+// GetNowTsCached getter for nowTs, if not set get now time and cache it
+func (sc *StatementContext) GetNowTsCached() time.Time {
+	if !sc.stmtTimeCached {
+		now := time.Now()
+		sc.nowTs = now
+		sc.stmtTimeCached = true
+	}
+	return sc.nowTs
+}
+
+// ResetNowTs resetter for nowTs, clear cached time flag
+func (sc *StatementContext) ResetNowTs() {
+	sc.nowTs = time.Time{}
```

Comment:
No need to perform this operation. It will be set when `GetNowTsCached()` is called.

Reply:
removed